authentication/authentication-internal-service/configure_start.sh

```diff
@@ -4,20 +4,11 @@
 set -e
 
 echo "Downloading pre-requisites install scripts"
-wget --no-check-certificate --no-cache --no-cookies $artifactory_url_env/artifactory/libs-release-local/deployment/docker/id-authentication/configure_biosdk.sh -O configure_biosdk.sh
```

Comment:
how the biosdk files are getting downloaded after removing download.

Reply:
As suggested by Sasi added biosdk directly in pom.